Sources/PointFree/Emails/EmailLayouts.swift

```diff
@@ -45,7 +47,7 @@ func simpleEmailLayout<A>(_ bodyView: View<A>) -> View<SimpleEmailLayoutData<A>>
             tr([
               td([align(.center), valign(.top)],
                  bodyView.view(layoutData.data)
-                  <> emailFooterView.view(unit))
+                  <> emailFooterView.view((layoutData.user, layoutData.newsletter)))
```

Comment:
we thread the user and newsletter down to the footer so that it can show info about why the user is receiving this email and how they can unsubscribe.